### ISSUE TITLE
Is it possible to get the framerate ?

### ISSUE THREAD

**Issue by markusmoenig** [parasyte/pixels#287]
Hi,

running my app on the web (based on the minimal_web) example gives me a 2 times higher framerate under Chrome than on Safari.

Is there a way to find out what the current fps is (or even to set it) ?

Thanks

**Comment by parasyte** [comment]
Wow I’m sorry this response is so late! I know I saw this question before but I thought I had already answered it.

what I usually do for FPS is compute the time delta between each draw call, and then average those samples over some duration (usually 1 second for obvious reasons).

There are a few ways to sample the time delta. I am typically interested in frame-to-frame timing, so I will do the sampling after the draw call returns.

it’s probably a good idea to put this into one of the demos!
